### PR TITLE
Add dip and refresh docker development environment

### PR DESCRIPTION
**Pull request by Envek**
`dip rspec` succeeds (that's unexpected!)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,52 +1,65 @@
...
+x-app-service-template: &app
+  image: anycable-demo:0.6.0
+  build:
+    context: .
+    dockerfile: .dockerdev/Dockerfile
+    args:
+      RUBY_VERSION: "2.5.7"
+      NODE_MAJOR: '12'
+      BUNDLER_VERSION: '1.17.1'
+      HIVEMIND_VERSION: '1.0.6'
```
Comment (palkan) [inline docker-compose.yml:12]: Why do we need hivemind in Docker 😃?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,52 +1,65 @@
...
-  backend: &backend
-    <<: *app
-    volumes:
-      - .:/app:cached
-      - assets:/app/public/assets
-    environment:
-      - RAILS_ENV=${RAILS_ENV:-development}
-      - REDIS_URL=redis://redis:6379/0
-      - CABLE_URL=ws://localhost:8080/cable
-      - ADAPTER=any_cable
```
Comment (palkan) [inline docker-compose.yml:22]: This param is missing in the new config (and that's what likely cause the exception in RPC service).

**Review by palkan — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (palkan) [reply docker-compose.yml:12]: Ok, I found it: we use it for specs.
And, actually, we can drop it, because `Profile.spec` contains only a single command.

Comment (Envek) [reply docker-compose.yml:12]: Here is also `Procfile.spec_erl` with two commands. Do we need it?

File: docker-compose.yml (hunk 2 above)
Comment (Envek) [reply docker-compose.yml:22]: Added and yes, it fixed the issue

File: docker-compose.yml (hunk 1 above)
Comment (palkan) [reply docker-compose.yml:12]: Nope. ErlyCable support is suspended, so we can drop everything related to it.

**Pull request merged**